### ISSUE TITLE
Uploads Directory

### ISSUE THREAD

**Issue by gpotter** [CareersSkillsIncubator/menteer#3]
Hello,
I see one of the last instructions for set up to be change permissions on /uploads.
I don't see an uploads folder.  Should I create one and where?
Thanks gang,
Can't wait to get into this app.

**Comment by eyalkatz** [comment]
Yes, you do have to create an /uploads directory in the main folder.

**Comment by gpotter** [comment]
This install is putting a whooping on me.  It just doesn't want to take.  As far as I can tell it's all set up right, db and files. I have redone the instructions like 3 times, but no luck.  CodeIgniter is functioning but I can't get a connection to my db.   There is probably something basic I'm missing.
www.esa.org/menteer/ 

Any advice?

Thanks,
-Gordon
